### PR TITLE
Hw3toconfirm

### PR DESCRIPTION
**Pull request by Raimund-zim**
Sorry, but  I haven't time(

### CODE REVIEW

File: CSS/body_style.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	height: 1026px;
+	position: absolute;
+	z-index: -2;
+}
+.banner {
+	margin-bottom: 0;
+	margin-left: auto;
+	margin-right: auto;
+
+	margin: 0 0 0 0;
```
Comment (shuliakleonid) [inline CSS/body_style.css:26]: margin may be short

File: CSS/body_style.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
+body {
+	width: 1510px;
+	margin-left: auto;
```
Comment (shuliakleonid) [inline CSS/body_style.css:3]: try this margin:0 auto

File: CSS/body_style.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	margin-right: auto;
+}
+.dark {
+	position: absolute;
+	width: 1510px;
+	height: 1026px;
+	margin-left: auto;
+	margin-right: auto;
```
Comment (shuliakleonid) [inline CSS/body_style.css:11]: margin: o auto

File: CSS/header_style.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+	list-style-type: none;
+}
+.logo__image {
+	margin: 0px 471.62px 0px 115.28px;
+}
+.header__info > li {
+	position: relative;
+	height: 75px;
+	width: 113.8px;
+	margin: 0 0 0 0;
```
Comment (shuliakleonid) [inline CSS/header_style.css:24]: short

File: CSS/regist__style.css [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+.start__button {
+	height: 60px;
+	width: 470px;
+	cursor: pointer;
+	background: #ec7979;
+	border: none;
+	border-radius: 0px 0px 10px 10px;
+	padding: 0px;
+	margin: 0px;
```
Comment (shuliakleonid) [inline CSS/regist__style.css:72]: matgin: 0 is valid too

**Review by shuliakleonid — APPROVED** [review]

File: HomeWork4/CSS/body_style.css [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+	height: 1026px;
+	position: absolute;
+	z-index: -2;
+}
+.banner {
+	margin-bottom: 0;
+	margin-left: auto;
+	margin-right: auto;
+
+	margin: 0 0 0 0;
```
Comment (shuliakleonid) [inline HomeWork4/CSS/body_style.css:26]: after all you add margin

**Review by shuliakleonid — APPROVED** [review]

**Review by Stalexus — APPROVED** [review]

File: CSS/body_style.css (hunk 1 above)
Comment (Raimund-zim) [reply CSS/body_style.css:26]: done

File: HomeWork4/CSS/body_style.css [hunk 7]
```diff
@@ -0,0 +1,27 @@
+body {
+	width: 1510px;
+	margin-left: auto;
+	margin-right: auto;
+}
```
Comment (shuliakleonid) [inline HomeWork4/CSS/body_style.css:5]: review that margin

File: HomeWork4/CSS/body_style.css [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+.dark {
+	position: absolute;
+	width: 1510px;
+	height: 1026px;
+	margin-left: auto;
+	margin-right: auto;
+	z-index: -1;
```
Comment (shuliakleonid) [inline HomeWork4/CSS/body_style.css:12]: do you need margin there

File: HomeWork4/CSS/body_style.css (hunk 6 above)
Comment (shuliakleonid) [reply HomeWork4/CSS/body_style.css:26]: delete unnecessary  margin

**Review by shuliakleonid — APPROVED** [review]

**Comment by Raimund-zim** [comment]
Done

**Comment by Raimund-zim** [comment]
Done

**Pull request merged**